task_DEV_4/task_DEV_4/Circle.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+namespace task_DEV_4
+{
+    class Circle : Figure
+    {
+        private const string MASSEGE_ERROR = "This value cannot be less than or equal to zero.";
```

Comment:
mEssAge

Reply:
less than or equal to zero? What does it mean?

Reply:
Need to add one empty line between fields and properties.

Reply:
What does `this value` mean? Maybe radius?

Reply:
done